src/s3access.cpp

```diff
@@ -168,7 +177,6 @@ QNetworkReply *S3Access::performOp(Method method, QString const &url, QString co
         break;
     }
 
-    free(digest);
 
```

Comment:
There's an extra line here after you've removed the `free(digest)`. If you could remove it, that would be appreciated.

Reply:
I don't see that line, I just removed that line here.

Reply:
Yes, I wasn't very clear. I mean if you have this:
```
1.
2. statement
3.
```
and then remove `statement`, you end up with two blank lines:
```
1.
3.
```
but for purely aesthetic reasons there should only be a single blank line after the switch statement.